x-pack/test/functional/page_objects/endpoint_page.ts

```diff
@@ -10,6 +10,24 @@ export function EndpointPageProvider({ getService }: FtrProviderContext) {
   const testSubjects = getService('testSubjects');
 
   return {
+    /**
+     * Finds the Data Grid with the given `selector` (test subject) and returns
+     * back an array containing the grid's header column text
+     *
+     * @param selector
+     * @returns Promise<string[]>
+     */
+    async dataGridHeaderVisibleText(selector: string) {
```

Comment:
Can you justify the usage of a 'page object'? Why not include this using regular typescript modules?